Cargo.toml

```diff
@@ -9,6 +9,6 @@ edition = "2021"
 ws = "0.9.2"
 serde = { version = "1.0.188", features = ["derive"] }
 serde_json = "1.0.107"
-shuttle-runtime = "0.37.0"
```

Comment:
I have no idea how this was set to `0.37.0`. I branched it off of main which is still using the old deps.